pennylane/interfaces/torch.py

```diff
@@ -23,7 +23,7 @@
 import pennylane as qml
 from pennylane.queuing import AnnotatedQueue
 
-COMPLEX_SUPPORT = semantic_version.match(">=1.6.0", torch.__version__)
+COMPLEX_SUPPORT = semantic_version.match(">=1.8.1", torch.__version__)

```

Comment:
Only to understand, is this because torch included better complex number support, which the device is using?

Reply:
Wait, I just saw Josh's explanation above.